Source/Core/VideoBackends/Vulkan/ObjectCache.cpp

```diff
@@ -244,12 +255,14 @@ bool ObjectCache::CreatePipelineLayouts()
        static_cast<u32>(compute_sets.size()), compute_sets.data(), 0, nullptr},
   }};
 
+  const bool ssbos_in_standard =
+      g_ActiveConfig.backend_info.bSupportsBBox || g_ActiveConfig.UseVSForLinePointExpand();
```

Comment:
`ssbos_in_standard` => `PIPELINE_LAYOUT_STANDARD` uses SSBOs

Reply:
After looking at this further, yeah, this is reasonable and fairly obvious from context. We might want to change `PIPELINE_LAYOUT_STANDARD` to `PIPELINE_LAYOUT_SPECIALIZED` or something like that since we've started referring to the opposite of ubershaders as "specialized shaders", but that's not something that needs to be handled in this PR.